src/main.tsx

```diff
@@ -5,14 +5,16 @@ import ReactDOM from 'react-dom';
 import { QueryClient, QueryClientProvider } from 'react-query';
```

Comment:
Components name convention requires using `uppercase` for making it easier to work with `React` library - you can find a bit about it in internet and `React` docs 😄

Reply:
Besides the `main` and `app` components seems to be a bit redundant right now. The scope of their task is so short, they could be merged IMO